### DISCUSSION TITLE
Build on MacOS

### DISCUSSION THREAD

**Discussion by fergdev** [MultiplatformKickstarter/Adoptme#2] [General]
Hey, love the project.

Just wondering why don't you use a MacOS box to build on for CI? 

I am having troubles building my own CMP project from the command line on MacOS, is there something that I don't know?

Also, I tried building this project from the command line and it hands on link:shared:linkReleaseFrameworkIosArm64. Just an FYI.

Love the project, thanks for the inspiration!

**Comment by ferranpons** [comment]
Hi @fergdev! Glad you like the project. 😊 

> Just wondering why don't you use a MacOS box to build on for CI?

I'm not using it right now because the template is not finished yet. I hope to have it soon.

> I am having troubles building my own CMP project from the command line on MacOS, is there something that I don't know?

I can't assyst with that. I'm sorry.

> Also, I tried building this project from the command line and it hands on link:shared:linkReleaseFrameworkIosArm64. Just an FYI.

It's not an issue, this is because I am using an Arm Macbook. If you are using an Intel Mac you would need to modify it on the "build.gradle.kts" file on the "shared" module.

Many thanks for your words!

**Reply by fergdev** [reply]
Thankyou. Looking forward to seeing how the project template develops.
